src/est/est_locl.h

```diff
@@ -181,25 +181,30 @@ DHE-DSS-AES256-SHA256"
 
 #define EST_HTTP_CT_PKCS7           "application/pkcs7-mime"
 #define EST_HTTP_CT_PKCS7_CO        "application/pkcs7-mime; smime-type=certs-only"
+#define EST_HTTP_CT_PKCS7_CO_LEN    45
 #define EST_HTTP_CT_CSRATTRS        "application/csrattrs"
 #define EST_HTTP_CE_BASE64          "base64"
+#define EST_HTTP_CE_BASE64_LEN      6
 #define EST_CSRATTRS_POP            "MAsGCSqGSIb3DQEJBw=="
-#define EST_CSRATTRS_POP_LEN         20
+#define EST_CSRATTRS_POP_LEN        20
 
 #define EST_HTTP_CT_MULTI           "multipart"
 #define EST_HTTP_CT_MULTI_LEN       9
 #define EST_HTTP_CT_MULTI_MIXED     "multipart/mixed"
 #define EST_HTTP_CT_MULTI_MIXED_LEN 15
 #define EST_HTTP_CT_PKCS8           "application/pkcs8"
+#define EST_HTTP_CT_PKCS8_LEN       17
 #define EST_HTTP_BOUNDARY           "est-server-boundary"
+#define EST_HTTP_BOUNDARY_LEN       19
+#define EST_HTTP_HDR_LEN_CONST      (EST_HTTP_BOUNDARY_LEN * 3) + (EST_HTTP_CE_BASE64_LEN * 2) + EST_HTTP_CT_PKCS8_LEN + EST_HTTP_CT_PKCS7_CO_LEN + 114
```

Comment:
I can't remember off the top of my head, but it looks like it is to cover the header keys and newlines.
eg. 
```
   HTTP/1.1 200 OK
   Status: 200 OK
   Content-Type: < xxx >
   Content-Length: < xxx >
```
Some of this could be calculated at run time and others could be a constant

Reply:
I have added this change